test/EVMHost.cpp

```diff
@@ -754,3 +754,17 @@ evmc::result EVMHost::resultWithGas(
 	result.output_size = _data.size();
 	return result;
 }
+
+void EVMHost::print_storage(ostringstream& _os)
+{
+	for (auto const& [addr, mockedAccount]: accounts)
+		print_storage(addr, _os);
+}
+
+void EVMHost::print_storage(evmc::address const& _addr, ostringstream& _os)
+{
+	_os << "Address: " << convertFromEVMC(_addr) << endl;
+	for (auto const& [slot, value]: accounts[_addr].storage)
```

Comment:
Does it maybe make more sense to make this ``get_storage`` and just have it return ``accounts[_addr].storage`` instead of directly printing it to an ``ostringstream`` (keeping the interface more general and letting the caller decide what to do with the data)?

Reply:
Done